generators/create-react-app/index.ts

```diff
@@ -15,16 +17,58 @@ class CreateReactAppGenerator extends PackageGenerator {
 
         switch (this.config.get('deployment')) {
             case DeploymentChoice.Ansible:
-                this.configureAnsible('ansible', {
+                this.configureAnsible('deployment/ansible', {
                     repositoryName: this.config.get('repositoryName'),
                 });
                 break;
             case DeploymentChoice.Kubernetes:
-                throw new Error('The create react app generator is not yet compatible with kubernetes deployment');
+                // Add files required for docker build
+                this.renderTemplate('deployment/kubernetes/docker', packagePath);
+
+                // Update chart
+                this.configureChart('deployment/kubernetes/chart');
+
+                this.replaceDestination(
+                    'modules/deployment/chart/values.yaml',
+                    /$/s,
+                    `\n${varify(packageName)}:\n  image:\n    tag: latest\n    digest: ~\n  sentry:\n    dsn: ~\n`,
+                );
+
+                this.writeTerraformVariable(
+                    `${varify(packageName)}_sentry_dsn`,
+                    'string',
+                    '"" # TODO Add sentry DSN here',
```

Comment:
Just an idea as I read this. 
Wouldn't it be great to store a list of "TODOs" to feedback at the end of the generation process so that users know what's remaining to do.